sql/hive-thriftserver/src/test/resources/log4j2.properties

```diff
@@ -27,10 +27,8 @@ appender.console.target = SYSTEM_OUT
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %d{HH:mm:ss.SSS} %p %c: %maxLen{%m}{512}%n%ex{8}%n
 
-appender.console.filter.1.type = Filters
-
-appender.console.filter.1.a.type = ThresholdFilter
-appender.console.filter.1.a.level = warn
+appender.console.filter.1.type = ThresholdFilter
```

Comment:
To avoid 

```
2025-11-20T05:56:22.1186895Z 2025-11-20T05:56:22.115648760Z pool-1-thread-1 ERROR Filters contains invalid attributes "onMatch", "onMismatch"
2025-11-20T05:56:22.1317678Z 2025-11-20T05:56:22.130070007Z pool-1-thread-1 ERROR Filters contains invalid attributes "onMatch", "onMismatch"
```